### DISCUSSION TITLE
How to open ContextMenu via long press on Android/iOS?

### DISCUSSION THREAD

**Discussion by alessandrofrancesconi** [AvaloniaUI/Avalonia#10490] [Q&A] [answered]
I'm building a multiplatform app that runs on Desktop, Android and iOS. I'm working with Avalonia 11.0.0-preview4.

I've added a ListBox whose items can be deleted using a ContextMenu, so I've written the following code:

```
<ListBox>
<!-- .... -->
  <ListBox.ItemContainerTheme>
    <ControlTheme TargetType="ListBoxItem" BasedOn="{StaticResource {x:Type ListBoxItem}}">
      <Setter Property="Padding" Value="8" />
      <Setter Property="ContextMenu">
        <Setter.Value>
          <ContextMenu>
            <MenuItem Header="Delete" Command="{Binding DeleteRecordCommand}"/>
          </ContextMenu>
        </Setter.Value>
      </Setter>
    </ControlTheme>
  </ListBox.ItemContainerTheme>
</ListBox>
```

On Windows 11, the menu appears via right-click, this is fine. On mobile, I would like to make it appear by "long press" over the listbox item. This currently does not work. How can I do it?

**Comment by maxkatz6** [comment, marked as answer]
It was implemented in preview5.

**Reply by alessandrofrancesconi** [reply]
Worked! Thanks
